tests/lib/rules/comma-spacing.js

```diff
@@ -81,26 +81,30 @@ ruleTester.run("comma-spacing", rule, {
         {code: "var arr = [1 ,2 ,3];", options: [{before: true, after: false}]},
         {code: "var obj = {'foo':'bar' , 'baz':'qur'};", options: [{before: true, after: true}]},
         {code: "var a = 1 , b = 2;", options: [{before: true, after: true}]},
-        {code: "var arr = [ , ];", options: [{before: true, after: true}]},
+        {code: "var arr = [, ];", options: [{before: true, after: true}]},
         {code: "var arr = [1 , ];", options: [{before: true, after: true}]},
         {code: "var arr = [ , 2];", options: [{before: true, after: true}]},
         {code: "var arr = [1 , 2];", options: [{before: true, after: true}]},
-        {code: "var arr = [ , , ];", options: [{before: true, after: true}]},
+        {code: "var arr = [, , ];", options: [{before: true, after: true}]},
         {code: "var arr = [1 , , ];", options: [{before: true, after: true}]},
         {code: "var arr = [ , 2 , ];", options: [{before: true, after: true}]},
         {code: "var arr = [ , , 3];", options: [{before: true, after: true}]},
         {code: "var arr = [1 , 2 , ];", options: [{before: true, after: true}]},
-        {code: "var arr = [ , 2 , 3];", options: [{before: true, after: true}]},
+        {code: "var arr = [, 2 , 3];", options: [{before: true, after: true}]},
         {code: "var arr = [1 , , 3];", options: [{before: true, after: true}]},
         {code: "var arr = [1 , 2 , 3];", options: [{before: true, after: true}]},
         {code: "a , b", options: [{before: true, after: true}]},
         {code: "var arr = [,];", options: [{before: false, after: false}]},
+        {code: "var arr = [ ,];", options: [{before: false, after: false}]},
         {code: "var arr = [1,];", options: [{before: false, after: false}]},
         {code: "var arr = [,2];", options: [{before: false, after: false}]},
+        {code: "var arr = [ ,2];", options: [{before: false, after: false}]},
```

Comment:
`before` is ignored in this case because there is no array element before the comma.